blocks/hero/hero.css

```diff
@@ -12,28 +12,85 @@ main .hero {
   position: relative;
   padding: 32px;
   min-height: 300px;
+  display: flex;
+  max-width: var(--max-content-width);
+  margin: auto;
+}
+
+main .hero .left-box {
```

Comment:
as discussed, I'd rename `left-box` and `right-box` to `hero-image-container` and `hero-content-container`